src/main/java/com/allocadia/carbonite/PersistedObjectCache.java

```diff
@@ -20,4 +24,14 @@
     public <T, I> T getObjectById(Class<T> clazz, I id, Function<I, T> fetcher) {
         return getInstanceCache(clazz, id).computeIfAbsent(Objects.requireNonNull(id), fetcher.andThen(Objects::requireNonNull));
     }
+
+    public <T extends PersistedObject> void addDirty(T po) {
```

Comment:
You're talking about po.setDirty?  I think it's probably fine having that public since it only gets added to the class as part of the weaving process...i.e. I don't think eclipse will show that method as being available

or am I misunderstanding?